### PR TITLE
[Docker][Onnx] Upgrade ONNX to latest version

### PR DESCRIPTION
**Pull request by AndrewZhaoLuo**
Allows us to support opset 14, 15, 16 in CI.

Add new unsupported tests to onnx frontened tests

### CODE REVIEW

**Comment by AndrewZhaoLuo** [comment]
cc @areusch

**Comment by areusch** [comment]
obnoxious, looks like a docs build failure. can you resubmit? then yes, we merge this and you should open a new issue to update docker images which outlines the rest of the process: https://github.com/apache/tvm/issues/new?assignees=&labels=&template=ci-image.md&title=%5BCI+Image%5D+

**Comment by leandron** [comment]
> obnoxious, looks like a docs build failure. can you resubmit? then yes, we merge this and you should open a new issue to update docker images which outlines the rest of the process: https://github.com/apache/tvm/issues/new?assignees=&labels=&template=ci-image.md&title=%5BCI+Image%5D+

I'm seeing that consistently on the nightly Docker image validation job, probably something we need to deal with separately.

**Comment by AndrewZhaoLuo** [comment]
cc @areusch plz merge

File: docker/install/ubuntu_install_onnx.sh [hunk 1]
```diff
@@ -27,9 +27,10 @@ set -o pipefail
 # https://github.com/onnx/onnx/pull/2834).  When updating the CI image
 # to onnx>=1.9, onnxoptimizer should also be installed.
 pip3 install \
-    onnx==1.8.1 \
-    onnxruntime==1.7.0
-
+    onnx==1.10.2 \
```
Comment (areusch) [inline docker/install/ubuntu_install_onnx.sh:30]: can you update these in `python/gen_requirements.py`? can be a follow-on.

**Review by areusch — APPROVED** [review]

**Comment by areusch** [comment]
thanks @AndrewZhaoLuo , can you update the gen_requirements.py in a follow-on?

**Pull request merged**